.github/workflows/scripts/monitor-typescript.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env node */
```

Comment:
We have node-ts, any reason we couldn't have written this as a typescript script?

Reply:
@evanpurkhiser I did not know we had it. I just looked at the other script in the gh folder. I'll make a PR to update it